### ISSUE TITLE
Detailed file view

### ISSUE THREAD

**Issue by hacdias** [hacdias/caddy-filemanager#58]
Suggested by @pepa65 on #56

**Comment by hacdias** [comment]
@pepa65, would you mind to try the new version (building from source)? There are some features that aren't available yet, but I'd like to know what do you think about the updated view modes. 😄

**Comment by pepa65** [comment]
This means I have to build filemanager and then build caddy with this repo?? I an try...

**Comment by hacdias** [comment]
Hello PePa,

You can find the instruction here:
https://github.com/mholt/caddy/wiki/Extending-Caddy. If you're having some
troubles you can tell me your OS and Architecture and I'll built it for you
:)

**Comment by pepa65** [comment]
Building it for me is probably going to be easier for you than explaining everything to me..! ;-)
I'm on Ubuntu 16.04 amd64

**Comment by hacdias** [comment]
Hey @PePa! I just left home now but I'll built it for you as soon as I
return home :) Thanks for the info!

**Comment by hacdias** [comment]
Here is it @pepa65 : https://www.dropbox.com/s/5b0y8f7mrpnd9ai/caddy_linux_amd64?dl=0 Sorry for the delay!

**Comment by hacdias** [comment]
Hey @pepa5, thanks for your opinion! Though, I'm not thinking about
changing the padding of the items.

Right now we support an external stylesheet. Just use the option
'stylesheet' as it is documented in the docs. You can do everything you
want with that stylesheet. You can even override completely the default
styles.

If you need help or something, tell me :)

**Comment by hacdias** [comment]
Hey @pepa65, I'll make a simple additional stylesheet you can use with this new version. I'll send it to you here when I'm done.

**Comment by hacdias** [comment]
@pepa65,

This styles:

```
#listing.list .item:not(.header) {
    padding: .3em 1em;
    font-size: .9em;
}

#listing.list .item:not(.header) div:first-of-type i {
    font-size: 1.5em;
    width: 2em;
}
```

Will produce this:

![image](https://cloud.githubusercontent.com/assets/5447088/21590309/a7afd464-d0f2-11e6-97c4-4f11ba96a4ae.png)

**Comment by hacdias** [comment]
On the new version, of course. I'm just going to close this issue because it is "done"/"finished". Though, if you have any more doubts about this, you can always contact me here or open a new issue.

**Comment by pepa65** [comment]
Excellent, thanks! I've settled on
```
#listing.list .item:not(.header){padding:.2em 1em; font-size:1em;}
#listing.list .item:not(.header) div:first-of-type i{font-size:1.3em; width:2em;}
```
Is this automatically going to be included in the next (0.9.5) build of caddy, or how does it work?? Right now with your build I can't use the filter plugin.

**Comment by hacdias** [comment]
The build I've sent you only has `filemanager` plugin. As soon as Matt updates everything on the server you'll be able to use the new version of File Manager which is not finished yet.

To use those styles, you'll have to have something like this:

```
filemanager {
     styles stylesheet.css
}
```

And put those lines on that file and it will work.
